21.1 mz dashboards eud (#135)
DevExpress/dotnet-eud: dashboard-for-web/articles/web-dashboard-designer-mode/ui-elements/dialogs-and-wizards/filter-editor.md

Then specify a comparison operator:

![Filtering_FilterEditor_ComparisonOperators](../../../../images/filtering_filtereditor_comparisonoperators132427.png)

Set an operand value type in the dedicated value box:

![Filtering_FilterEditor_SelectOperand](../../../../images/filtering_filtereditor_selectoperand132419.png)

The following operand types are available:

* **Value** - Allows you to compare dimension and static values.
	
	![Filtering_FilterEditor_ValueDropdown](../../../../images/filtering_filtereditor_valuedropdown132420.png)
* **Property** - Compares different dimension values.
	
	![Filtering_FilterEditor_FieldsDropdown](../../../../images/filtering_filtereditor_fieldsdropdown132425.png)
* **Parameter** - Allows you to compare dimension and [dashboard parameter](../../data-analysis/dashboard-parameters.md) values.
	
	![Filtering_FilterEditor_ParameterDropdown](../../../../images/filtering_filtereditor_parameterdropdown132421.png)

Click the filter condition's **Remove** ![web-filter-editor-remove-button](../../../../images/web-rd-filter-editor-remove-button129484.png) button to delete the condition.

## Advanced Mode

> The **Advanced Mode** allows you to enter a custom filter string.

Reviewer: ```suggestion
**Advanced Mode** allows you to enter a custom filter string.
```

![Filtering_FilterEditor_TextMode](../../../../images/filtering_filtereditor_textmode132423.png)

Consider the following syntax conventions when you create text-based filter conditions:

* Insert a dimension by enclosing its name in square brackets (for example, **[Category]**).
* Denote string values with apostrophes (for example, **'Bikes'**).
* Enclose date-time values with hashtags (for example, **#2019-06-01#**).
* Reference [dashboard parameters](../../data-analysis/dashboard-parameters.md) by adding a question mark before their names (for example, **[Category] = ?categoryParam**)

> This editor supports intelligent code completion (suggesting functions, parameters, and available data columns as you type).

Reviewer: ```suggestion
This editor supports intelligent code completion (which suggests functions, parameters, and available data columns as you type).
```

![Filtering_FilterEditor_TextMode_Autocompletion](../../../../images/filtering_filtereditor_textmode_autocompletion132424.png)

> You can add a comment to your expression to explain it and make the expression more readable. Comments are multi-line and begin with `/*` and end with `*/`.

Reviewer: Are all comments multi-line? Or did you mean that comments _can be_ multi-line? For example:

```suggestion
You can add a comment to your expression to explain it and make the expression more readable. Comments can be multi-line, and begin with `/*` and end with `*/`.
```

The **Warning** ![expression-editor-error-icon](../../../../images/expression-editor-error-icon118339.png) icon appears if a condition contains errors.

> ## OLAP Filtering Specifics

Reviewer: ```suggestion
## OLAP Filter Specifics
```
